app/components/button/styles/NavigateIconButtonStyle.js

```diff
@@ -0,0 +1,21 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    position: 'relative',
+    width: '100%',
+    height: 80,
+    borderBottomWidth: 2,
+    flexDirection: 'row',
+  },
+  icon: {
+    width: 50,
+    height: 50,
+    left: 15,
+    top: 15,
+  },
+  text: { fontSize: 24, top: 28, left: 25 },
```

Comment:
We need to start using the function moderateScale() because expecially for the font it won't look as great on other devices